package.json

```diff
@@ -27,6 +27,9 @@
     "q": "1.4.1",
     "rx": "4.1.0"
   },
+  "peerDependencies": {
+    "fh-mbaas-api": "7.0.0-3"
```

Comment:
I see that the `latest-7` dist-tag on npmjs points to 7.0.0-4. Is there anything important in that newer build that we should have here?